web/packages/teleport/src/components/FormLogin/FormIdentifierFirst.tsx

```diff
@@ -0,0 +1,294 @@
+              onNotYou={onNotYou}
+            />
+          )}
+        </Flex>
+      </Flex>
+    </Card>
+  );
+}
+
+function UsernamePrompt({
+  onSubmitUsername,
+  username,
+  setUsername,
+  fetchAttempt,
+}: {
+  onSubmitUsername(): void;
+  username: string;
+  setUsername: (username: string) => void;
+  fetchAttempt: Attempt<void>;
+}) {
+  const inputRef = useRefAutoFocus<HTMLInputElement>({
+    shouldFocus: true,
+  });
+
+  return (
+    <Flex
+      as="form"
+      alignItems="center"
+      justifyContent="center"
+      flexDirection="column"
+      onSubmit={e => {
+        e.preventDefault();
+        onSubmitUsername();
+      }}
+      width="100%"
+      gap={3}
+      mb={3}
+      mt={5}
+    >
+      <Flex flexDirection="column" alignItems="flex-start" width="100%">
+        <LabelInput>Username</LabelInput>
+        <Input
+          ref={inputRef}
+          value={username}
+          onChange={e => setUsername(e.target.value?.trim())}
+          placeholder="Username or email address"
+          width="100%"
+          autoFocus={true}
+        />
+      </Flex>
+      <ButtonPrimary
+        type="submit"
+        size="extra-large"
+        disabled={fetchAttempt.status === 'processing' || !username}
+        width="100%"
+      >
+        Next
+      </ButtonPrimary>
+    </Flex>
+  );
+}
+
+function ConnectorList({
+  providers,
+  onLoginWithSso,
+  onNotYou,
+}: {
+  providers: AuthProvider[];
+  onLoginWithSso(provider: AuthProvider): void;
+  onNotYou(): void;
+}) {
+  const $btns = providers.map((item, index) => {
+    let { name, type, displayName } = item;
+    const title = displayName || name;
+    const ssoType = guessProviderType(title, type);
+    return (
+      <StyledButtonSso
+        px={5}
+        size="extra-large"
+        key={index}
+        title={`Sign in with ${title}`}
+        ssoType={ssoType}
+        onClick={e => {
+          e.preventDefault();
+          onLoginWithSso(item);
+        }}
+      />
+    );
+  });
+
+  return (
+    <Flex flexDirection="column" gap={2} mt={5} width="100%">
+      <Text textAlign="start" typography="subtitle1" color="text.muted">
+        Select an identity provider to continue:
+      </Text>
+      {$btns}
+      <Flex justifyContent="center" mt={3}>
+        <ViewSwitchButton onClick={onNotYou} disabled={false}>
+          Sign in with a different account
+        </ViewSwitchButton>
+      </Flex>
+    </Flex>
+  );
+}
+
+/**
+ * ViewSwitch button is the button used to switch between login form views.
+ */
+export function ViewSwitchButton({
+  onClick,
+  disabled,
+  children,
+}: {
+  onClick(): void;
+  disabled: boolean;
+  children: React.ReactNode;
+}) {
+  return (
+    <ButtonText
+      size="large"
+      onClick={onClick}
+      disabled={disabled}
+      css={`
+        color: ${props => props.theme.colors.text.muted};
+        background: transparent;
+        &:hover {
+          background: transparent;
+        }
+      `}
```

Comment:
Do the `bg="transparent"` (or maybe `background="transparent"`) and `color="text.muted"` props not work here?

Reply:
No since `bg`, `background`, and `color` aren't valid props since they're not valid HTML attributes on `button` HTML elements and aren't part of our `ButtonProps` either, so I had to use inline-styling here